test/test_config_converter.py

```diff
@@ -0,0 +1,47 @@
+"""
+File to run tests for the config_converter algorithm
+
+Usage: python3 -m pytest
+Note: Run this file from the parent directory (outside test folder)
+"""
+
+import subprocess
+import config_converter.config_converter as config_converter
+
+
+def read_file(path):
+    with open(path, 'rt') as f:
+        return f.read()
+
+
+def check_equality(config_name):
+    fluentd_path = f'test/data/original/{config_name}.conf'
```

Comment:
[Optional suggestion] Consider making the folder as `case name`, and put both original and expected in the same directory. This way, it's easier to spot the same usecases within the same folder.

Reply:
I like this suggestion - made the folder!